presto-hive/src/main/java/com/facebook/presto/hive/HiveErrorCode.java

```diff
@@ -55,7 +55,8 @@
     HIVE_INVALID_BUCKET_FILES(28, EXTERNAL),
     HIVE_EXCEEDED_PARTITION_LIMIT(29, USER_ERROR),
     HIVE_WRITE_VALIDATION_FAILED(30, INTERNAL_ERROR),
-    HIVE_PARTITION_DROPPED_DURING_QUERY(31, EXTERNAL);
+    HIVE_PARTITION_DROPPED_DURING_QUERY(31, EXTERNAL),
+    HIVE_TABLE_READ_ONLY(32, USER_ERROR);
```

Comment:
Let's format it like we do in `StandardErrorCode`:
```java
HIVE_TABLE_READ_ONLY(32, USER_ERROR),
/**/;
```
This allows for a cleaner diff when adding new ones, since you don't have to touch the last one.